vcpkg/get_started/get-started.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Install and use packages with vcpkg
+description: Tutorial guides the user through the process of installing and using packages with vcpkg.
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: operating-systems
+author: JavierMatosD
+ms.author: javiermat
+ms.topic: tutorial
+ms.date: 08/23/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to install and manage packages using vcpkg, so that I can easily set up and maintain C++ projects with necessary dependencies.
+---
+
+# Tutorial: Install and use packages with vcpkg
+
+This tutorial shows you how to create a C++ "Hello World" program that uses the `fmt` library with CMake and vcpkg. You'll install dependencies, configure, build, and run a simple application.
+
+## Prerequisites
+
+- A terminal to work in.
+- A C++ compiler
```

Comment:
On Windows, this will fail unless the user _specifically_ has Visual Studio installed. I think we should mention that somehow.